src/WebJobs.Extensions.OpenAI/Agents/ChatBotService.cs

```diff
@@ -271,6 +271,9 @@ public async Task PostMessageAsync(ChatBotPostRequest request, CancellationToken
                     chatState.Metadata.TotalMessages);
             }
 
+            // Set the total tokens that have been consumed.
+            chatState.Metadata.TotalTokens = response.Value.Usage.TotalTokens;
```

Comment:
missed during migration to table storage, added the validation to tests to avoid in future